baseplate/server/__init__.py

```diff
@@ -415,11 +422,15 @@ def load_and_run_shell() -> None:
         setup = _load_factory(config.shell["setup"])
         setup(env, env_banner)
 
+    configure_logging(config, False)
```

Comment:
Should this be `configure_logging(config, debug=args.debug)` to enable debug logging when requested?

Reply:
yup, thought `args` didn't exist in that function, but it's loaded up above, so fixing this